Started new planner stuff

gpennell opened the pull request.

ZaneBartlett1 commented:
https://github.com/DartExplore/dallas-by-metro-frontend/pull/86/commits/17047de6e058497ff3ef6575aed543867d977d3a closes #83 and #81

gpennell committed:
Initial commit of new planner

Note: Zane and Garrett both contributed substantially to this work, but the commit history has been rewritten.
